client/src/SignUpShopper.js

```diff
@@ -0,0 +1,98 @@
+import { useEffect, useState } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import Api from './Api';
+
+//to add new component, make new file
+//to create (crud), use forms
+function SignUpShopper() {
+
+    //:id from Sections.js 
+    const {id} = useParams();
+    const history = useHistory();
+    
+    //initial value of section (in this case, obj with name, slug, and position attributes)
+    const [shopper, setShopper] = useState({
+        UserId: 0,
+        zipcode: 0
+    });
+
+    const [user, setUser] = useState({
+      photo: '',
+      birthday: 0,
+      gender: ''
+    });
+
+    //side effects, don't directly interact with output, don't refresh when it changes
+    useEffect(function(){
+        if(id){
+            Api.shoppers.get(id).then((response) => setShopper(response.data));
+            Api.users.get(id).then((response) => setUser(response.data));
```

Comment:
If you look in client/src/Api.js you'll notice that there isn't actually a Api.users.get(id) function implemented, but there is a Api.users.me() function. This is what I was talking about before... the "me" route calls /api/users/me which returns the logged in user (without having to specify it by id).

I'll help you create a similar route method for shoppers which you can then use here.